### PR TITLE
Invalidate sessions

### PR DESCRIPTION
**Pull request by ricardoamoreno**
## Summary

- /users/sessions/invalidate_all endpoint added
- sessionExpire field added to user model
- sessionExpireDate verification added on token
- invalidate_sessions tests added

## Known Issues

Issue in validateToken.js
Possible race condition: `req.user` might be reassigned. 
partial solution:  eslint-disable-next-line require-atomic-updates. 

## Trello Card

[Deshabilitar todas las sesiones](https://trello.com/c/vznJsZTT)

### CODE REVIEW

File: app/controllers/users.js [hunk 1]
```diff
@@ -45,3 +47,15 @@ exports.adminUser = async (req, res, next) => {
     next(error);
   }
 };
+
+exports.invalidateSessions = async (req, res, next) => {
+  try {
+    const { user } = req;
+    const session = await invalidateSessionsInteractor(user);
+    logger.info(session);
+    res.status(200).send({ message: sessionClosedMessage });
```
Comment (martinremis) [inline app/controllers/users.js]: This could better be a 204 without body

File: app/interactors/users.js [hunk 2]
```diff
@@ -52,3 +57,9 @@ exports.createAdminUserInteractor = async (user, body) => {
   logger.info(`admin user ${adminUser[0].id} was ${action} succesfully`);
   return { userId: adminUser[0].id };
 };
+
+exports.invalidateSessionsInteractor = async user => {
+  const userSession = await updateUser(user.id, { sessionExpire: Date.now() });
+  if (!userSession[0]) throw notFoundError(userNotFoundErrorMessage);
+  return { userId: userSession[1][0].id };
```
Comment (martinremis) [inline app/interactors/users.js]: Why do you return this? I think there is no need of return something here

**Review by martinremis — CHANGES_REQUESTED** [review]
Well done! Just some comments

File: app/controllers/users.js (hunk 1 above)
Comment (ricardoamoreno) [reply app/controllers/users.js]: got it

File: app/interactors/users.js (hunk 2 above)
Comment (ricardoamoreno) [reply app/interactors/users.js]: it's true, I deleted it

**Review by martinremis — APPROVED** [review]

**Pull request merged**